include/settings.fxml

```diff
@@ -1,47 +1,76 @@
-<?import javafx.scene.layout.GridPane?>
-<?import javafx.scene.layout.HBox?>
-<?import javafx.scene.layout.RowConstraints?>
+<?import javafx.geometry.*?>
+<?import javafx.scene.control.*?>
+<?import javafx.scene.layout.*?>
 
-<GridPane hgap="10.0" maxHeight="-Infinity" maxWidth="-Infinity" minHeight="-Infinity" minWidth="-Infinity" xmlns="http://javafx.com/javafx/18" xmlns:fx="http://javafx.com/fxml/1" fx:controller="me.mrletsplay.shittyauthlauncher.ShittyAuthSettingsController">
-  <columnConstraints>
-    <ColumnConstraints hgrow="SOMETIMES" />
-    <ColumnConstraints hgrow="SOMETIMES" minWidth="10.0" />
-  </columnConstraints>
-  <rowConstraints>
-      <RowConstraints minHeight="10.0" prefHeight="30.0" vgrow="SOMETIMES" />
-      <RowConstraints minHeight="10.0" prefHeight="30.0" vgrow="SOMETIMES" />
-      <RowConstraints minHeight="10.0" prefHeight="30.0" vgrow="SOMETIMES" />
-      <RowConstraints minHeight="10.0" prefHeight="30.0" vgrow="SOMETIMES" />
-    <RowConstraints minHeight="10.0" prefHeight="30.0" vgrow="SOMETIMES" />
-  </rowConstraints>
-   <children>
-      <Label text="Always patch authlib" GridPane.rowIndex="1" />
-      <CheckBox fx:id="checkboxAlwaysPatchAuthlib" mnemonicParsing="false" GridPane.columnIndex="1" GridPane.rowIndex="1" />
-      <HBox alignment="CENTER_RIGHT" spacing="10.0" GridPane.columnIndex="1" GridPane.hgrow="NEVER" GridPane.rowIndex="4" GridPane.vgrow="NEVER">
-         <children>
-            <Button mnemonicParsing="false" onAction="#buttonCancel" text="Cancel" />
-            <Button mnemonicParsing="false" onAction="#buttonSave" text="Save">
-               <HBox.margin>
-                  <Insets />
-               </HBox.margin>
-            </Button>
-         </children>
-      </HBox>
-      <Label text="Always patch minecraft.jar" GridPane.rowIndex="2" />
-      <CheckBox fx:id="checkboxAlwaysPatchMinecraft" mnemonicParsing="false" GridPane.columnIndex="1" GridPane.rowIndex="2" />
-      <Label text="Minimize launcher after starting the game" GridPane.rowIndex="3" />
-      <CheckBox fx:id="checkboxMinimizeLauncher" mnemonicParsing="false" GridPane.columnIndex="1" GridPane.rowIndex="3" />
-      <Label text="Use JRE downloads from Adoptium instead of the ones provided by Mojang" />
-      <CheckBox fx:id="checkboxUseAdoptium" mnemonicParsing="false" GridPane.columnIndex="1" />
-   </children>
-   <padding>
-      <Insets bottom="10.0" left="10.0" right="10.0" top="10.0" />
-   </padding>
-</GridPane>
+<AnchorPane maxHeight="-Infinity" maxWidth="-Infinity" minHeight="-Infinity" minWidth="-Infinity" xmlns="http://javafx.com/javafx/18" xmlns:fx="http://javafx.com/fxml/1" fx:controller="me.mrletsplay.shittyauthlauncher.ShittyAuthSettingsController">
+    <TabPane layoutX="6.0" layoutY="5.0" tabClosingPolicy="UNAVAILABLE" AnchorPane.bottomAnchor="0" AnchorPane.leftAnchor="0.0" AnchorPane.rightAnchor="0.0" AnchorPane.topAnchor="0.0">
+        <tabs>
+            <Tab text="Settings">
```

Comment:
Maybe this should be called `General`, because a `Settings` tab inside the settings menu is kind of weird